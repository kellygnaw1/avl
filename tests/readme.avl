# Ignore this line and the next one.

# Print ranges that would cause a left-left rotation
print left-left

# Print ranges that would cause a right-left rotation
print right-left

print left-right

print right-right
