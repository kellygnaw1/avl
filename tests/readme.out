No inserts would cause a left-left rotation.

No inserts would cause a right-left rotation.

No inserts would cause a left-right rotation.

No inserts would cause a right-right rotation.

